code/modules/client/client_procs.dm

```diff
@@ -1,3 +1,8 @@
+var/list/localhost_addresses = list(
+	"127.0.0.1" = TRUE,
+	"::1" = TRUE
```

Comment:
BYOND doesn't support IPv6, so this is kinda pointless.

Reply:
It's also harmless though. Call it future proofing.